osc-server/src/main/java/org/osc/core/broker/service/GetDtoFromEntityService.java

```diff
@@ -132,7 +132,7 @@
             res.setDto((R) dto);
         } else if (entityName.equals("Appliance")) {
             Appliance entity = getEntity(entityId, entityName, Appliance.class, em);
-            ApplianceDto dto = new ApplianceDto();
+            ApplianceDto dto = new ApplianceDto(null, null, null);
```

Comment:
passing null, null, null in constructor is probably a bad idea. Since this is a valid case where dto is initialized to null...lets just use the public contructor of ApplianceDto